test/Microsoft.DocAsCode.Build.Common.Tests/ModelAttributeHandlerTest.cs

```diff
@@ -364,6 +391,8 @@ public class SimpleModel
             public string Identity { get; set; }
             [UniqueIdentityReference]
             public List<object> Identities { get; set; }
+            [Href]
```

Comment:
yes... renamed to UrlContent

---
In reply to: [111536147](https://github.com/dotnet/docfx/pull/1543#discussion_r111536147) [](ancestors = 111536147)